libraries/incubator/ORM/Entity/EntityBuilder.php

```diff
@@ -385,11 +382,12 @@ public function handleStorage(Element $storage)
 					{
 						case 'csv':
 							$handler = '\Joomla\ORM\Storage\CsvProvider';
-							$param   = $parts[1];
+							break;
+						case 'orm':
+							$handler = '\Joomla\ORM\Storage\Doctrine\DoctrineProvider';
```

Comment:
There is nothing in the way for the API provider to get more information, which then should be provided using tags within `<api>...</api>`